Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -33,19 +32,25 @@ struct ASPanelUX {
 class ActivityStreamPanel: UITableViewController, HomePanel {
     weak var homePanelDelegate: HomePanelDelegate? = nil
     private let profile: Profile
-    private var onyxSession: OnyxSession?
+    private let telemetry: ActivityStreamTracker
+
     private let topSitesManager = ASHorizontalScrollCellManager()
     private var isInitialLoad = true //Prevents intro views from flickering while content is loading
     private let events = [NotificationFirefoxAccountChanged, NotificationProfileDidFinishSyncing, NotificationPrivateDataClearedHistory, NotificationDynamicFontChanged]
 
+    private var sessionStart: Timestamp?
+
     lazy var longPressRecognizer: UILongPressGestureRecognizer = {
         return UILongPressGestureRecognizer(target: self, action: #selector(ActivityStreamPanel.longPress(_:)))
     }()
 
     var highlights: [Site] = []
 
-    init(profile: Profile) {
+    init(profile: Profile, telemetry: ActivityStreamTracker? = nil) {
```

Comment:
I've added a nil default parameter there to allow me to pass in a mocked version of the tracker in the test cases. This doesn't change any of the existing callsites in the app.